lib/generators/rodauth/templates/app/views/rodauth/tailwind/change_login.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :title, rodauth.change_login_page_title %>
+<div class="dark:bg-gray-900 flex items-center justify-center h-screen">
+  <div class="max-w-sm">
+    <h3 class="mb-3 text-3xl font-semibold text-center dark:text-gray-200"> Change Your Login Email Addres </h3>
+    <%= form_with url: rodauth.change_login_path, method: :post, data: { turbo: false } do |form| %>
+      <div class="mx-4 py-1">
+        <!-- The label should probably be named "email"  under rodauth's settings-->
+        <%= form.label "login", "Email", class: "block text-sm text-gray-400 dark:text-gray-100" %>
+        <%= form.email_field rodauth.login_param, value: params[rodauth.login_param], id: "login", autocomplete: "email", required: true, class: class_names("w-full px-3 py-2 border rounded-md border-gray-400 dark:border-gray-700 dark:bg-gray-900 dark:text-gray-100 focus:dark:border-emerald-400 focus:dark:bg-gray-800 invalid:border-red-500 invalid:text-red-600 focus:invalid:border-red-500 focus:invalid:ring-red-500 valid:border-green-500", {"border rounded-md border-red-500": rodauth.field_error(rodauth.login_param)}), aria: ({ invalid: true, describedby: "login_error_message" } if rodauth.field_error(rodauth.login_param)) %>
+        <p class="text-xs text-gray-600 dark:text-gray-300 mt-1"> Enter new email address</p>
```

Comment:
Would the form still look good without this? I would like to avoid using any hardcoded english text, so that the whole form is translated when used with rodauth-i18n.

Reply:
Fixed here: [2c6b4fe](https://github.com/janko/rodauth-rails/pull/114/commits/2c6b4fef8b8bc9b20ec9eac159b033e237b63a56)

On this point, I'll throw in my two cents:

![image](https://user-images.githubusercontent.com/15097447/192899646-2aa068f2-b715-4bcb-b7f1-39f4ccf23007.png)